src/handbook/company/security/cryptography.md

```diff
@@ -0,0 +1,81 @@
+---
+navTitle: Cryptography Policy
+---
+
+# Cryptography Policy
+
+| Policy owner   | Effective date |
+| -------------- | -------------- |
+| @knolleary     | 2023-06-01     |
+
+## Purpose
+
+To ensure proper and effective use of cryptography to protect the confidentiality,
+authenticity and/or integrity of information. This policy establishes requirements
+for the use and protection of cryptographic keys and cryptographic methods
+throughout the entire encryption lifecycle.
+
+## Scope 
+
+All FlowForge data, information and information systems that handle confidential
+or critical information.
+
+## Cryptography Policy
+
+FlowForge requires all team members apply appropriate cryptographic controls in
+handling of data. Where encryption is in use, strong cryptography with associated
+key management processes and procedures shall be implemented. All encryption
+shall be performed in accordance with industry standards, including NIST SP 800-57.
+
+Customer or confidential company data must utilize strong ciphers and configurations
+in accordance with vendor recommendations and industry best practices including
+[NIST when stored or transferred over a public network](https://csrc.nist.gov/projects/cryptographic-standards-and-guidelines). 
+
+
+FlowForge requires all team members to comply with the cryptography policy, such that:
+
+ - All Production Data at rest is stored on encrypted volumes.
```

Comment:
Please add to the onboarding checklist that disk encryption should be turned on for all devices that will store FlowForge Data

Reply:
We will have a requirement to install the Vanta agent as part of the onboarding for any FF-owned devices. This will ensure disk encryption is enabled.

The onboarding checklist will include a vanta step that will cover all of this. Once we have more policies in place, I will use someone in the team as a first walk through of the vanta onboarding, and add appropriate pieces to our checklist.